### PR TITLE
Split up actor entity ID functions

### PR DESCRIPTION
**Pull request by KeyMaster-**
#### Description
Added the option to either get an int64 entity ID from an actor, or a convenience method to get an FString directly.

#### Release note
REQUIRED: Add a release note to the `##Unreleased` section of CHANGELOG.md. You can find guidance for writing useful release notes [here](../SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md). Documentation changes are exempt from this requirement.

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?

STRONGLY SUGGESTED: How can this be verified by QA?

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp [hunk 1]
```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
...
-FString USpatialStatics::GetActorEntityIDString(const AActor* Actor)
+int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
 	if (Actor != nullptr)
 	{
 		if (const USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Actor->GetNetDriver()))
 		{
-			const Worker_EntityId EntityId = SpatialNetDriver->PackageMap->GetEntityIdFromObject(Actor);
-			return FString::Printf(TEXT("%lld"), EntityId);
+			return (int64)SpatialNetDriver->PackageMap->GetEntityIdFromObject(Actor);
```
Comment (samiwh) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:182]: Use a static_cast

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp [hunk 2]
```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
 	PrintStringSpatial(WorldContextObject, InText.ToString(), bPrintToScreen, TextColor, Duration);
 }
 
-FString USpatialStatics::GetActorEntityIDString(const AActor* Actor)
+int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
 	if (Actor != nullptr)
```
Comment (samiwh) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:178]: Should this be a check? Do we have valid reasons for wanting to support nullptr?

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp [hunk 3]
```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
...
 	{
 		if (const USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Actor->GetNetDriver()))
```
Comment (samiwh) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:180]: I'd flip this around and return early if it's nullptr if we want this to be valid. If we don't have a valid reason for calling this when we don't have a net driver should we also just make this a check?

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp (hunk 2 above)
Comment (KeyMaster-) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:178]: A check seems reasonable

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp (hunk 3 above)
Comment (KeyMaster-) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:180]: Check on actor and net driver then, good catch.

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp [hunk 4]
```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
...
 }
 
-FString USpatialStatics::GetActorEntityIDString(const AActor* Actor)
+int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
-	if (Actor != nullptr)
+	check(Actor);
+	check(Actor->GetNetDriver());
+
+	if (const USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Actor->GetNetDriver()))
```
Comment (samiwh) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp:181]: We don't need a check and an if.

**Pull request merged**